ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSplashScreenView.swift

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+struct FocusSplashScreenView: View {
+  var namespace: Namespace.ID
+
+  @State private var isShimmering = false
+
+  var body: some View {
+    NavigationView {
+      GeometryReader { geometry in
+        ZStack {
+          braveLogo
+            .scaleEffect(isShimmering ? 1.015 : 1.0)
+            .overlay(
+              self.linearGradientView
+                .frame(width: geometry.size.width, height: 2 * geometry.size.height)
+                .position(
+                  x: geometry.size.width / 2,
+                  y: isShimmering ? 2 * geometry.size.height : -geometry.size.height
+                )
+                .scaleEffect(isShimmering ? 1.015 : 1.0)
+                .mask(
+                  braveLogo
+                )
+            )
+          Image("focus-icon-brave", bundle: .module)
+            .resizable()
+            .matchedGeometryEffect(id: "icon", in: namespace)
+            .frame(width: 146, height: 146)
+        }
+        .background(Color(braveSystemName: .pageBackground))
+        .onAppear {
+          withAnimation(.linear(duration: 1.25)) {
+            self.isShimmering = true
+          }
+        }
+      }
+    }
+    .navigationViewStyle(StackNavigationViewStyle())
+    .navigationBarHidden(true)
+  }
+
+  private var braveLogo: some View {
+    Image("focus-logo-brave", bundle: .module)
+      .resizable()
+      .aspectRatio(contentMode: .fit)
+      .frame(maxWidth: .infinity, maxHeight: .infinity)
+      .padding(.leading, 42)
+  }
+
+  private var linearGradientView: some View {
+    Rectangle()
+      .foregroundColor(.clear)
+      .background(
+        LinearGradient(
+          stops: [
+            Gradient.Stop(color: Color(UIColor(rgb: 0xFFFFFF)).opacity(0), location: 0.00),
```

Comment:
If you update the Leo/Figma npm package, does this gradient get added so that you don't need to hardcode these colours and locations?

Reply:
Yeah this is a one of gradient that doesnt exist in design system. 🤷